### PR TITLE
CC - Initial commit

### PR DESCRIPTION
**Pull request by nMoncho**
Still WIP, just wanted to upload the schema, a readme and a short example.
Before submitting the final PR, will upload the whole Scala Knowledge graph.

### CODE REVIEW

File: core-curriculum/schema/knowledge-graph.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {},
+  "id": "lunatech-core-curriculum-knowledge-graph-schema",
+  "properties": {
```
Comment (nMoncho) [inline core-curriculum/schema/knowledge-graph.json]: Why not some URL/path?

Comment (eamelink) [inline core-curriculum/schema/knowledge-graph.json]: Ah yes, I didn't mean to put the SVG in the json, a path would be fine there :)

File: core-curriculum/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- A technology, such as HTML.
+
+## Schema
+Knowledge graphs are based on a schema to provide a shared and common structure, we decided on JSON schema since it will allow us to use `git` as a means to contribute.
+
+## Contributing
+Please feel free to contribute to this project, the easiest way to get started is to take a look at the schema and some instances of such. At this present moment gathering material should be our focus and the most welcomed contribution.
+
+Each subject's knowledge graph is defined in a separate file, in which is stated:
+- Id: An unique string identifier for the subject.
```
Comment (nMoncho) [inline core-curriculum/README.md]: Done :)

File: core-curriculum/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Name and description: Subjet's name and description.
```
Comment (nMoncho) [inline core-curriculum/README.md]: Done :)

File: core-curriculum/schema/knowledge-graph.json (hunk 1 above)
Comment (nMoncho) [inline core-curriculum/schema/knowledge-graph.json]: Done :)
Added **image** property to subject's schema.

**Pull request merged**